rollup.config.js

```diff
@@ -3,9 +3,12 @@ import sourcemaps from 'rollup-plugin-sourcemaps';
 import {plugins} from './build/rollup_plugins';
 
 const version = JSON.parse(fs.readFileSync('package.json')).version;
-
-const production = process.env.BUILD === 'production';
-const outputFile = production ? 'dist/mapbox-gl.js' : 'dist/mapbox-gl-dev.js';
+const {BUILD} = process.env;
+const minified = BUILD === 'production';
+const production = BUILD === 'production' || BUILD === 'production-unminified';
```

Comment:
Can we remove the conflation of debug/release and minification? We could have two separate variables: `BUILD` controls dev vs. release, while `UGLIFY` (defaulting to true) controls minification